ee/agent/startup/startup.go

```diff
@@ -0,0 +1,94 @@
+// Package startup provides access to and manages storage of startup data:
+// flags/config values/settings that launcher needs during initialization,
+// before the knapsack is available.
+package startup
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	_ "github.com/golang-migrate/migrate/v4/database/sqlite"
+	"github.com/kolide/launcher/ee/agent/flags/keys"
+	agentsqlite "github.com/kolide/launcher/ee/agent/storage/sqlite"
+	"github.com/kolide/launcher/ee/agent/types"
+	_ "modernc.org/sqlite"
+)
+
+// GetStartupValue retrieves the value for the given flagKey from the startup database
+// located in the given rootDirectory. It wraps creation and closing of the sqlite store.
+func GetStartupValue(ctx context.Context, rootDirectory string, flagKey string) (string, error) {
```

Comment:
```suggestion
// GetValue retrieves the value for the given flagKey from the startup database
// located in the given rootDirectory. It wraps creation and closing of the sqlite store.
func GetValue(ctx context.Context, rootDirectory string, flagKey string) (string, error) {
```

I'm not yet convinced we want to expose this here, vs through knapsack, but if we do we can use a simpler signature. Even just `Value` I guess

Reply:
I will rename these when I rename the package 🙂